.changeset/silly-garlics-live.md

```diff
@@ -0,0 +1,60 @@
+---
+'astro': minor
+---
+
+Built-in View Transitions Support (experimental)
+
+Astro now supports [view transitions](https://developer.chrome.com/docs/web-platform/view-transitions/) through the new `<ViewTransitions />` component and the `transition:animate` (and associated) directives. View transitions are a great fit for content-oriented sites, and we see it as the best path to get the benefits of client-side routing (smoother transitions) without sacrificing the more simple mental model of MPAs.
+
+In this release, view transitions are enabled by add the experimental flag to your config:
+
+```js
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  experimental: {
+    viewTransitions: true,
+  },
+})
+```
+
+This enables you to use the new APIs added.
+
+## <ViewTransitions />
+
+This is a component which acts as the *router* for transitions between pages. You need to add it to each page, in the `<head>` section, in which you want to have transitions away from be done in-client. To enable support throughout your entire app, add the component in some common layout or component.
+
+__CommonHead.astro__
+
+```astro
+---
+import { ViewTransitions } from 'astro:transitions';
+---
+
+<meta charset="utf-8">
+<title>{Astro.props.title}</title>
+<ViewTransitions />
+```
+
+With only this change, your app will now route completely in-client. You can then add transitions to individual elements using the `transition:animate` directive.
+
+## Animations
+
+Using `transition:animate` you can use some of our built-in animations. For example:
+
+```astro
+<header transition:animate="slide">
+```
+
+Will perform an animation in which the element slides out to the left, and slides in from the right. You can also customize these animations by specifying a duration like so:
```

Comment:
``````suggestion
Add `transition:animate` to any element to use Astro's built-in animations.

```astro
<header transition:animate="slide">
```

In the above, Astro's `slide` animation will cause the `<header>` element to slide out to the left, and then slide in from the right when you navigate away from the page.

You can also customize these animations using any CSS animation properties, for example, by specifying a duration:
``````

We try to keep descriptions of different code samples on their own lines, so that we don't describe both the above and below together!

Check my wording here. It's a little more verbose but I do like explicitly mentioning:
- *add this to an element* vs "use" so that usage is as clear as possible
- showing off that you can add any valid CSS! (Check my wording: "any CSS animation properties" <- is that exactly correct, or the best way to say it?)